authors/settings/development.py

```diff
@@ -0,0 +1,27 @@
+# Database
+# https://docs.djangoproject.com/en/1.11/ref/settings/#databases
+
+from authors.settings.base import *
```

Comment:
What is this one importing exactly?

Reply:
There's a base.py file in the settings folder that has a lot of the common stuff that's used by development.py and testing.py. So, I am importing all that e.g. INSTALLED APPS etc.

Reply:
@AGMETEOR I see one line in base.py and when after the import, I don't see it used anywhere right now.

Reply:
One line?

Reply:
@AGMETEOR , never mind. I have seen it!!

Reply:
Alright.